### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by 7rulnik**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,14 +60,14 @@
...
-    "karma-mocha-reporter": "^1.1.5",
+    "karma-mocha-reporter": "^2.0.0",
     "karma-safari-launcher": "^0.1.1",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.7.0",
     "lint-staged": "^0.1.1",
     "minimist": "^1.2.0",
     "mocha": "^2.3.4",
-    "ngrok": "0.1.99",
+    "ngrok": "2.1.6",
```
Comment (mxstbr) [inline package.json]: ngrok **has** to stay on > v2.0, otherwise tunnelling won't work anymore!

Comment (7rulnik) [inline package.json]: Did you meant less then 2.0?
I try start tunnel on 2.1.6 — it's work.

Comment (mxstbr) [inline package.json]: Yeah, less than 2.0.

You probably have an out-of-date version in your `node_modules`. As of `2.0.0`, ngrok requires an account and authentication – see #177. We've made that mistake before! :see_no_evil:

Comment (7rulnik) [inline package.json]: Take a look on this issue — https://github.com/bubenshchykov/ngrok/issues/34#issuecomment-195614093
I want to try reinstall node_modules and try again. You made me doubt :D

Comment (mxstbr) [inline package.json]: Oooh, nice, good catch! :+1:

**Comment by mxstbr** [comment]
Thanks a lot @7rulnik!

**Pull request merged**